rust/src/wallet.rs

```diff
@@ -254,7 +254,7 @@ impl Wallet {
     }
 }
 
-fn get_wallet() -> Result<MutexGuard<'static, Wallet>> {
+pub fn get_wallet() -> Result<MutexGuard<'static, Wallet>> {
```

Comment:
nit: it will be safer if we won't leak the `MutexGuard` outside of this file - I will remove the `pub` in a follow-up. :)